### ISSUE TITLE
False positive in Moq1302: Invalid member 'StatusCodes.Status202Accepted' in LINQ to Mocks expression 

### ISSUE THREAD

**Issue by abatishchev** [rjmurillo/moq.analyzers#1010]
Code like this:

```csharp
var blobClient = new Mock<BlobClient>();
blobClient.Setup(static c => c.ExistsAsync(It.IsAny<CancellationToken>()))
          .ReturnsAsync(Response.FromValue(false, Mock.Of<Response>(static r => r.Status == StatusCodes.Status200OK)));
````

is flagged by Moq1302:

> StorageClientTests.cs(104,104): warning Moq1302: Invalid member 'StatusCodes.Status202Accepted' in LINQ to Mocks expression (https://github.com/rjmurillo
/moq.analyzers/blob/2bb7f0eed5f3133058840986f4c0e92082e83446/docs/rules/Moq1302.md) [D:\Projects\ATS-RP\src\
Provisioning\StorageClient.Tests.Unit\StorageClient.Tests.Unit.csproj]

What's wrong with `StatusCodes.Status202Accepted`? The code compilers and runs well.
